spec/models/validators/date_beyond_osra_establishment_validator_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+describe DateBeyondOsraEstablishmentValidator do
```

Comment:
`RSpec.describe` is generally better to use than `describe` for the first call to describe.  If we ever turn off the Rails monkey-patching to increase app performance with `config.expose_dsl_globally= false` then this will be needed. `RSpec.describe` does everything `describe` does, plus more, and (I think) it is more readable what is going on.